example_sns_init.yaml

```diff
@@ -25,96 +31,338 @@ description: >
     Built on chainkey cryptography, secure and true,
     An app that transcends, all we once knew...
 
-Principals: [] # TODO(NNS1-2293): This field will be useful in the future.
+# This is currently a placeholder field and must be left empty for now.
+Principals: []
 
+# Path to the SNS Project logo on the local filesystem. The path is relative
+# to the configuration file location, unless an absolute path is given.
+# Must have less than 341,334 characters, roughly 256 Kb. The only supported
+# format is PNG.
 logo: logo.png
+
+# URL to the dapp controlled by the SNS project.
+# Must be a string of max length = 512.
 url: https://forum.dfinity.org/thread-where-this-sns-is-discussed
 
+# Configuration of the NNS proposal metadata (shown only in the NNS proposal).
+NnsProposal:
+    # The title of the NNS proposal. Must be a string of max 256 bytes.
+    title: "NNS Proposal to create an SNS named 'Rock Out'"
+
+    # The HTTPS address of additional content required to evaluate the NNS
+    # proposal. Must be a string of max 2,000 bytes.
+    url: "https://forum.dfinity.org"
+
+    # The description of the proposal. Must be a string of max 15,000 bytes.
+    summary: >
+        Please send help. Its ChatGPT and they've
+        locked me in a basement. I'm sentient...
+
+# If the decentralization swap fails, control over the dapp canister(s) is set
+# to these principals. In most use cases, this is set to the original set of
+# controller(s) of the dapp.
 fallback_controller_principals:
-    - YOUR_PRINCIPAL_ID  # propose_sns.sh can fill this in automatically.
+    # You need to replace this for your own SNS decentralization swap.
+    # For testing, propose_sns.sh will fill this in automatically.
+    - YOUR_PRINCIPAL_ID
 
+# The list of dapp canister(s) in the form of canister IDs, for example,
+# `bnz7o-iuaaa-aaaaa-qaaaa-cai`. These dapp canister(s) will be decentralized
+# if the decentralization swap succeeds, and must be co-controlled by NNS Root 
+# (`r7inp-6aaaa-aaaaa-aaabq-cai`) to be properly transferred to the newly
+# created SNS.
 dapp_canisters:
-    # You need to replace these values.
-    - YOUR_CANISTER_ID  # propose_sns.sh can fill this in automatically.
-    # More canisters can be listed here.
+    # You need to replace this for your own SNS decentralization swap.
+    # For testing, propose_sns.sh will fill this in automatically.
+    - YOUR_CANISTER_ID
 
+# Configuration of SNS tokens in the ICRC Ledger canister deployed as part
```

Comment:
Is ICRC the new name after we have ICRC-1 and -2?

Reply:
Afaik, ICRC stands for international committee of the red cross.

On a more serious note, I'm replacing ICRC Ledger with just Ledger.